src/settings.js

```diff
@@ -519,6 +519,10 @@ var GL_SUPPORT_EXPLICIT_SWAP_CONTROL = 0;
 // you can turn it off.
 var GL_POOL_TEMP_BUFFERS = 1;
 
+// Specifies the size of the GL temp buffer pool, in bytes. Must be a multiple
+// of 9 and 16.
+var GL_POOL_TEMP_BUFFERS_SIZE = 2*9*16; // == 288
```

Comment:
does this need to be a public setting?

looks like this also changes the value, which was 256 - was that intentional?

Reply:
Changing the value is intentional, note that the check changed from `if (x*9 <= GL_POOL_TEMP_BUFFERS_SIZE)` to `if (x <= GL_POOL_TEMP_BUFFERS_SIZE/9)` to save code size. That requires a value that is multiple of 9 and 16.

It does not need to be a public setting, but I don't think we have a good way for creating file-specific "#defines"?

Reply:
sg about the value.

About the setting, this could at least be an internal setting and not a public one. But we can actually set defines in files, {{{ X = Y }}} at the top of the file will just assign that value in the compiler's global scope.

Reply:
That's neat - I think I tried something like that, but botched up the syntax. Moved GL_POOL_TEMP_BUFFERS_SIZE to library_webgl.js now.